test/integration/admin_dashboard_test.rb

```diff
@@ -34,4 +35,29 @@ class AdminDashboardTest < ActionDispatch::IntegrationTest
     refute page.has_content?('Welcome, Default_user!')
     assert page.has_content?("The page you were looking for doesn't exist.")
   end
+
+  test 'admin can send an email to all users with daily deal' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.